Managers/UInputMan.cpp

```diff
@@ -134,6 +133,39 @@ namespace RTE {
 				return true;
 			}
 		}
+#elif defined __unix__
+		int numDetectedJoysticks = 0;
+		std::string devName = "/dev/input/jsX";
+		std::string devNameFallback = "/dev/jsX";
+		for (numDetectedJoysticks = 0; numDetectedJoysticks < MAX_JOYSTICKS; ++numDetectedJoysticks) {
```

Comment:
I don't love how this is done, but I guess you have to have the string to check if it exists, and reusing it is better than building a new one each time. Probably isn't any reasonable way to do this better :(

Reply:
rebuilding the string every time adds no measurable overhead (less than microseconds, menu updates are in the milliseconds) so this is much cleaner